core/new-gui/src/app/dashboard/component/feature-container/user-project-list/user-project-list.component.ts

```diff
@@ -64,21 +65,54 @@ export class UserProjectListComponent implements OnInit {
       });
   }
 
-  public removeEditStatus(pid: number): void {
-    this.userProjectEntriesIsEditingName = this.userProjectEntriesIsEditingName.filter(index => index != pid);
+  public removeEditStatus(pid: number, editType: string): void {
+    switch (editType) {
+      case "name":
+        this.userProjectEntriesIsEditingName = this.userProjectEntriesIsEditingName.filter(index => index != pid);
+      case "description":
+        this.userProjectEntriesIsEditingDescription = this.userProjectEntriesIsEditingDescription.filter(
+          index => index != pid
+        );
```

Comment:
I think it's better to separate them, use a method to explicitly handle each case.

Reply:
Sounds good, rewrote to separate them